commerce-subscription-web/src/main/java/com/liferay/commerce/subscription/web/internal/display/context/CommerceSubscriptionEntryDisplayContext.java

```diff
@@ -417,7 +418,7 @@ protected String getNavigation() {
 	private final CommerceSubscriptionEntryService
 		_commerceSubscriptionEntryService;
 	private final ConfigurationProvider _configurationProvider;
-	private final CPRequestHelper _cpRequestHelper;
+	private final CommerceSubscriptionRequestHelper _cpRequestHelper;
```

Comment:
`CommerceSubscriptionRequestHelper _cpRequestHelper` should be `CommerceSubscriptionRequestHelper _commerceSubscriptionRequestHelper`